rolelst.c

```diff
@@ -47,11 +60,13 @@ static int parse_options(int argc, char **argv, int *numeric_flag)
 }
 
 int main(int argc, char **argv) {
+    struct rolelst_settings settings;
+    settings.numeric_flag = 0;
```

Comment:
Nitpick: you'd better
```c
memset(&settings, 0,sizeof(settings));
```
This guarantees that all members of struct will be zero initialized no matter what the exact definition of the  `struct rolelst_settings` is.